src/libServer/LookupServer.cpp

```diff
@@ -1459,6 +1458,16 @@ Json::Value LookupServer::GetTransactionsForTxBlock(const string& txBlockNum) {
 
   auto const& txBlock = m_mediator.m_txBlockChain.GetBlock(txNum);
 
+  return GetTransactionsForTxBlock(txBlock,
+                                   m_mediator.m_lookup->m_historicalDB);
+}
+
+Json::Value LookupServer::GetTransactionsForTxBlock(const TxBlock& txBlock,
```

Comment:
Any reason for adding this extra level of function call (`GetTransactionsForTxBlock(txBlock) -> GetTransactionsForTxBlock(txBlock, historicalDB)`)?

Reply:
Making it static so that can be used as a utility function for WebsocketServer